vignettes/efa-score.Rmd

```diff
@@ -116,8 +116,8 @@ Recover factor covariances with 2S-PA
 ts_fit <- tspa("",
```

Comment:
Turns out I originally used the empty model (`""`) trick to get a default model with all covariances. Maybe we consider keeping this and instead check whether the model is a character variable? Thanks

Reply:
Sounds good. `tspa()` now only check if the model is a character variable.